test/e2e/run.sh

```diff
@@ -104,6 +104,7 @@ while getopts ":a:RpS:B:s:gjCJH:wzyl:cm:f:iIUvxu:h:F" opt; do
       ;;
     g)
       MAGELLAN_CONFIG="magellan.json"
+      MOCHA_ARGS+="-file lib/mocha-hooks/mocha-hooks.js"
```

Comment:
This is a (hopefully) temporary solution kept only as long as CircleCI tests are run.